templates/uspec.erb

```diff
@@ -11,7 +11,7 @@ else
     t_tag_spec.push('NOEXEC')
 end
 
-if ['RedHat','CentOS'].include?(@facts['operatingsystem'])
+if ['RedHat'].include?(@facts['osfamily'])
```

Comment:
This is not specific enough without an associated `simplib::assert_metadata()` in the code.